grc/blocks/variable_struct.block.yml.py

```diff
@@ -17,21 +17,21 @@
         self.${{id}} = ${{id}} = struct({{
             % for i in range({0}):
             <%
-                field = getVar('field' + str(i))
+                field = context.get('field' + str(i))
             %>
             % if len(str(field)) > 2:
-            ${{field}}: getVar('value' + str(i)),
+            ${{field}}: context.get('value' + str(i)),
```

Comment:
I see `${field}` in my entry box.hhhhhh

Reply:
Rebuilding to test, but I pushed fixes to what I think is correct.  I am a little unclear what "var_value" is for so I am not sure why the field key is getting assigned to field.  Looks like there was a bug #2007 opened a while ago with someone else confused about what it is for.